### ISSUE TITLE
[14] Pricelist report is not compatible with Multi UoM

### ISSUE THREAD

**Issue by sswapnesh** [odoo/odoo#59690]
Impacted versions: 14.0

Steps to reproduce: 

Create a Product with Different Unit of Measure
![Screenshot from 2020-10-10 15-04-47](https://user-images.githubusercontent.com/16624719/95651726-f5b7b280-0b09-11eb-81bc-4d13d72c1a5b.png)

Generate Pricelist
![Screenshot from 2020-10-10 15-05-12](https://user-images.githubusercontent.com/16624719/95651737-023c0b00-0b0a-11eb-9ddd-fce2ded7d87d.png)

Current behavior: Unit of Measure is not considered.
![Screenshot from 2020-10-10 15-05-35](https://user-images.githubusercontent.com/16624719/95651753-108a2700-0b0a-11eb-84fa-ae5be5345245.png)

Expected behavior: It should Consider Unit of Measure. (It should be $ 1 for 1 Unit)

Video/Screenshot link (optional):

**Comment by Feyensv** [comment]
Pricelists have no concept of UoM, the value they express is always given based on the product UoM.
The value expressed in the columns are in Units of the product UoM.

E.g: a service product sold in hours will be shown as **price/Hour**, not price/Unit, it has no sense at all.

Therefore, the behavior is correct IMO.  But we could eventually add an UoM column to display the respective products uom's on the rows.

**Comment by Yenthe666** [comment]
I'd close this as not a bug honestly :)

**Comment by pedrobaeza** [comment]
I agree is not a bug, but the label is very confusing. It would better to have there "1 <uom>" instead of "1 Units". The same apply to products with other UoMs like liters, meters, etc. Would you prefer to have "1 Unit" = x €, or "1 Meter(s)" = x €?

**Comment by Feyensv** [comment]
It would require either grouping the data by product uoms or to add the information on each row clearly.
I doubt this would be an acceptable change for a "stable" version, even if I agree it's a bit confusing.

**Comment by Feyensv** [comment]
I'm not convinced it would really be better :confused:.

Let's keep it like this, IMO it's still fundamentally correct, even if unit doesn't strictly refer to the "Unit" uom.

**Comment by pedrobaeza** [comment]
OK, I prefer my proposal to be more neutral, but as you think it's better.

**Comment by dnplkndll** [comment]
is this related to the concept of you might have a price list on a `multiple` of a carton versus a min of an each? For items you have carton prices a simple UOM on the price list would allow you to at least use the same sku to price and pull versus a kit setup to hole multiple value on the price list item 

Looking for the best way to address items with unit of pack 100 in one price. each another (breaking carton price) so 150 each should be priced at a 1 carton of 100 and 50 each price.

**Comment by Smkapasi** [comment]
I sell goods to supermarkets and i sell the same product to them in different UOM prices in the same Invoice.

Example 
Single pen = 1$ 
a pack of 3 pen = 2.5$
a pack of 5 pen = 4$

I need a pricelist that picks the prices as per the UOM.
